docs/more-pushmsg.md

```diff
@@ -83,3 +83,36 @@ switch ($code) {
         echo "<strong>Hmm Something Went Wrong or HTTP Status Code is Missing</strong>";
 }
 ```
+
+### Node.js - Using Axios
+
+```javascript
+////////////////////////////////////////
+// Send Push Message to Gotify Server //
+////////////////////////////////////////
+
+var url = "http://localhost:8008/message?token=<apptoken>";
+var bodyFormData = {
+  title: "Hello from Javascript",
+  message: "Test Push Service from Node.js",
+  priority: 5
+};
+axios({
+  method: "post",
+  headers: {
+    "Content-Type": "application/json"
+  },
+  url: url,
+  data: bodyFormData
+})
```

Comment:
Please either use semicolons throughout the file, or don't use them at all. I personally prefer omitting them.

Reply:
actually it calls from the above values

```js
var url = "http://localhost:8008/message?token=<apptoken>";
var bodyFormData = {
  title: "Hello from Javascript",
  message: "Test Push Service from Node.js",
  priority: 5
};
```

Reply:
To be consistent you either add a semicolon here as end of this statement, or omit all semicolons from this script.

Reply:
sorry just not clearly noticed let me change

Reply:
`npm run format` removes the semicolon